### PR TITLE
New schema figure

### PR DESCRIPTION
**Pull request by ivirshup**
Updating schema figure to be one from the paper. Currently having trouble building anndata docs locally due to something with the pytorch stuff.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
 
 # default settings
 templates_path = ["_templates"]
+html_static_path = ["_static"]
```
Comment (ivirshup) [inline docs/conf.py:33]: I'm not sure if this actually does anything

**Comment by ivirshup** [comment]
I would like the schema to sit on the right side of the page, an allow text to be on the same level.

File: README.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@
 .. |Coverage| image:: https://api.codacy.com/project/badge/Coverage/b92ae35b691141ceb5f2ee74beaf39d3
    :target: https://www.codacy.com/manual/theislab/anndata
 
-anndata - Annotated Data
+anndata - Annotated data
```
Comment (falexwolf) [inline README.rst:20]: See: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings

File: docs/index.rst [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 .. image:: _static/img/anndata_schema.svg
    :align: right
-   :width: 350px
+   :width: 260px
```
Comment (falexwolf) [inline docs/index.rst:6]: This should not have broken the docs.

File: anndata/_core/anndata.py [hunk 4]
```diff
@@ -138,18 +138,15 @@ class AnnData(metaclass=utils.DeprecationMixinMeta):
     variables :attr:`var` (:attr:`varm`, :attr:`varp`),
     and unstructured annotations :attr:`uns`.
 
-    .. figure:: https://falexwolf.de/img/scanpy/anndata.svg
-       :width: 350px
+    .. figure:: _static/img/anndata_schema.svg
```
Comment (falexwolf) [inline anndata/_core/anndata.py]: This might have broken the docs.

File: docs/conf.py (hunk 1 above)
Comment (flying-sheep) [reply docs/conf.py:33]: I think it needs to be set. We didn’t have files, so didn’t need it

File: anndata/_core/anndata.py [hunk 5]
```diff
@@ -138,18 +138,15 @@ class AnnData(metaclass=utils.DeprecationMixinMeta):
     variables :attr:`var` (:attr:`varm`, :attr:`varp`),
     and unstructured annotations :attr:`uns`.
 
-    .. figure:: https://falexwolf.de/img/scanpy/anndata.svg
-       :width: 350px
+    .. figure:: ../_static/img/anndata_schema.svg
+       :width: 260px
```
Comment (flying-sheep) [inline anndata/_core/anndata.py:142]: the font looks a bit wonky for me:

grafik

Comment (ivirshup) [reply anndata/_core/anndata.py:142]: The PDF looks fine though, could probably go from pdf -> svg

**Comment by ivirshup** [comment]
I think 40% looks consistently good, especially since the central container has it's max width capped

**Comment by falexwolf** [comment]
Sounds good!

**Review by falexwolf — APPROVED** [review]

**Comment by ivirshup** [comment]
Oh, I still need to update the svg with one where the font isn't weird

**Comment by ivirshup** [comment]
It turns out the issue was too few decimal points for the coordinates in the SVG – so the new version will be 1000x more precise

Not so scalable when you encode numbers as text I guess

**Comment by flying-sheep** [comment]
> Not so scalable when you encode numbers as text I guess

hilariously, some JSON files are actually smaller than some binary variations of the same data, since each ASCII character is only 8 bit, so `3.2` is 24 bit and `1` is 8 bit, but both are 32 bit in the smallest common fixed-sized integer representation.

**Pull request merged**